internal/shim-kvm/src/hostcall.rs

```diff
@@ -492,9 +492,9 @@ impl Handler for HostCall<'_> {
         Ok(())
     }
 
-    fn getrandom(&mut self, buf: &mut [u8], flags: c_uint) -> sallyport::Result<size_t> {
-        if flags & !(GRND_NONBLOCK | GRND_RANDOM) != 0 {
-            return Err(EINVAL);
+    fn getrandom(&mut self, buf: &mut [u8], flags: c_uint) -> sallyport::Result<c_size_t> {
+        if flags & !(libc::GRND_NONBLOCK | libc::GRND_RANDOM) != 0 {
+            return Err(libc::EINVAL);
```

Comment:
Same issue as @rvolosatovs noticed here.

Reply:
I've favored a policy of bringing these items into scope.